daffodil-cli/src/it/scala/org/apache/daffodil/blob/TestBlob.scala

```diff
@@ -64,15 +53,13 @@ class TestBlob {
       // Use +7 to drop the 'file://' to get the path of the file
       val generated_blob = result.substring(result.indexOf("file://") + 7).takeWhile(_ != '<')
 
-      //val checksum = String.format("%s parse -s %s %s", Util.binPath, testSchemaFile, testInputFile)
-      val cmp = String.format("/usr/bin/cmp %s %s", generated_blob, testInputFile)
-      shell.sendLine(cmp)
-      shell.expect(contains("\n"))
-      shell.sendLine("echo $?")
-      shell.expect(contains("0\n"))
+      md5sum(shell, generated_blob, "a8430718a3dc7cab13d28c8838e298c4")
 
       // Clean up blobs
-      shell.sendLine("rm -rf daffodil-blobs")
+      if (Util.isWindows)
+        shell.sendLine("rmdir /Q /S daffodil-blobs")
+      else
+        shell.sendLine("rm -rf daffodil-blobs")
```

Comment:
Thoughts on putting this is a finally block? If something fails we could be leaving around some giant files.

Also suggest moving this to a utility function.